config/locales/shared/proportion_protected/en.yml

```diff
@@ -7,5 +7,6 @@ en:
       title_present: Confirmed Presence
       title_absent: Confirmed Absence
       title_unknown: Not Known
+      title_data_deficient: Data Deficient
```

Comment:
Is the duplication here necessary? Is it needed in countries/shared, if it's in shared?